policyengine_us/parameters/gov/states/me/tax/income/deductions/itemized/cap.yaml

```diff
@@ -0,0 +1,12 @@
+description: Maine itemized deductions before medical and dental expenses are capped at this amount.
+values:
+  2022-01-01: 31_150
```

Comment:
could you also include the 2021 value so we can check it with Martin against Taxsim
and source

Reply:
could you also change the start dates for the [start and width](https://github.com/PolicyEngine/policyengine-us/tree/master/policyengine_us/parameters/gov/states/me/tax/income/deductions/phase_out) parameters to [2018](https://www.mainelegislature.org/legis/statutes/36/title36sec5125.html)